libraries/rush-lib/src/logic/operations/ShellOperationRunner.ts

```diff
@@ -19,6 +19,7 @@ export interface IOperationRunnerOptions {
   rushProject: RushConfigurationProject;
   rushConfiguration: RushConfiguration;
   commandToRun: string;
+  commandForHash: string;
```

Comment:
For the cache ID?

Reply:
For the `getConfigHash()` function, which in turn powers the cache id, yeah.